lib/pbkit/pbkit.c

```diff
@@ -2234,7 +2234,7 @@ void pb_stop_log(void)
 #endif
 
 
-void pb_end(DWORD *pEnd)
+DWORD *pb_end(DWORD *pEnd)
```

Comment:
This function should also check the correctness of `pEnd` (same conditions as for `pb_pushto` input apply: it must be at the expected location + the expected `pb_PushNext` location should probably be made invalid so it's not used errornously).

Reply:
I believe this has been taken care of now.